### DISCUSSION TITLE
텍스트 레이아웃 목록과 스크롤바 사이에 여백

### DISCUSSION THREAD

**Discussion by da-in** [Yangjaecheon-Otter-Guardians/simple-thumbnail#132] [Bugs]
### Bugs Description

```bash
텍스트 레이아웃 목록과 스크롤바 사이에 여백이 있으면 좋을 것 같습니다.
4px 정도인지 디자이너님 자문을 구하면...🥺
```

### BDD TASK

```bash
given - 화면에 진입한다.
then - 레이아웃 목록과 스크롤바 사이에 여백이 있다.
```

**Comment by caseBread** [comment]
저거 스크롤바 없애는건 별로겠죠..? 

유저입장에서 저 잘린부분을 통해 가로스크롤 할수있겠다는걸 인지할수도 있지 않을까 싶어서욥

아니면 여백 추가하는것도 좋겠습니다 :) 스크롤바가 붙어있으니까 어색한것 같기도 하네용

<img width="645" alt="스크린샷 2024-02-18 오후 8 09 19" src="https://github.com/Yangjaecheon-Otter-Guardians/simple-thumbnail/assets/92029332/29bd2838-fde3-4d95-94a4-a23e4642be5a">

**Reply by da-in** [reply]
확실히 미관상 없는게 깔끔하니 취향이지만🥺
화면 사이즈에 따라 블럭이 잘리지 않는 경우가 발생할 수도 있다고 생각해서, 명시적으로 앞뒤 선택지가 더 있다는 걸 보여주는 게 좋은 것 같습니다.
이 부분은 @se-chive 와 논의가 필요할 것 같습니다!
